### PR TITLE
Check that heuristics script is passing on cluster

### PR DESCRIPTION
**Pull request by alvarocarvajald**
Check that heuristics script is passing on at least one node on the output of `crm corosync status qnetd`.

- Related ticket: N/A
- Needles: N/A
- Verification run: [node 1](http://mango.suse.de/tests/2385), [node 2](http://mango.suse.de/tests/2386), [qnetd server](http://mango.suse.de/tests/2387), [support server](http://mango.suse.de/tests/2384)

### CODE REVIEW

File: tests/ha/qnetd.pm [hunk 1]
```diff
@@ -29,19 +29,20 @@ sub qdevice_status {
     $num_nodes-- if ($expected_status eq 'stopped');
 
     # Check qdevice status
-    script_run "$qnetd_status_cmd";
+    $output = script_output "$qnetd_status_cmd", proceed_on_failure => 1;
```
Comment (alvarocarvajald) [inline tests/ha/qnetd.pm:32]: We could also remove the `, proceed_on_failure => 1` and instead use `if ($expected_status ne 'stopped')`. @ldevulder @juadk what do you think would be better?

Comment (juadk) [reply tests/ha/qnetd.pm:32]: Maybe `if ($expected_status ne 'stopped')` is more readable.

Comment (ldevulder) [reply tests/ha/qnetd.pm:32]: If it works yes, it's more readable.

Comment (alvarocarvajald) [reply tests/ha/qnetd.pm:32]: OK. Will push this change and add a new verification run.

**Comment by alvarocarvajald** [comment]
New verification runs:

- Node 1: http://mango.suse.de/tests/2389
- Node 2: http://mango.suse.de/tests/2390
- Qnetd Server: http://mango.suse.de/tests/2391
- Support Server: http://mango.suse.de/tests/2388

**Pull request merged**